security/us-east-1/security-hub/config.tf

```diff
@@ -0,0 +1,23 @@
+#=============================#
+# AWS Provider Settings       #
+#=============================#
+provider "aws" {
+  region  = var.region
+  profile = var.profile
+}
+
+#=============================#
+# Backend Config (partial)    #
+#=============================#
+terraform {
+  required_version = "~> 1.2"
+
+  required_providers {
+    aws = "~> 4.10"
+  }
+
+  backend "s3" {
+    key = "legacy/us-east-1/security-hub/terraform.tfstate"
```

Comment:
Sure you have to use `legacy` here?

Reply:
We took it from a real case where legacy was used. I agree with changing this to the account name here in the refarch.

Reply:
Also, in the case we grab it from we used region, but we don't use to use it in the refarch. What about this @diego-ojeda-binbash ?

Reply:
Good catch. Let's just adhere to our conventions. Not much to decide on here.

Reply:
fixed